Exiled.Events/Patches/Generic/PlayerHasHintPatch.cs

```diff
@@ -0,0 +1,69 @@
+// -----------------------------------------------------------------------
+// <copyright file="PlayerHasHintPatch.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.Events.Patches.Generic
+{
+    using System.Collections.Generic;
+    using Exiled.API.Features;
+    using HarmonyLib;
+    using Hints;
+    using MEC;
+
+    /// <summary>
+    /// Patches <see cref="HintDisplay.Show(Hint)"/>.
+    /// </summary>
+    [HarmonyPatch(typeof(HintDisplay), nameof(HintDisplay.Show))]
+    public static class PlayerHasHintPatch
+    {
+#pragma warning disable SA1313 // Parameter names should begin with lower-case letter
+#pragma warning disable SA1600 // Elements should be documented
+#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
+        // Creating a list for coroutine check
+        private static List<Player> plyHaveCoroutine = new List<Player>();
+
+        public static void Postfix(HintDisplay __instance, Hint hint)
+        {
+            // Try to get the player, if it doesn't exist, just return
+            Player player = Player.Get(__instance.gameObject);
+            if (player == null)
```

Comment:
You should use a shorter form: `if (!(Player.Get(__instance.gameObject) is Player player))`